eslint.config.js

```diff
@@ -129,33 +130,31 @@ module.exports = [
     {
         files: ["lib/rules/*", "tools/internal-rules/*"],
         ignores: ["**/index.js"],
-        rules: {
-            ...eslintPlugin.configs["rules-recommended"].rules,
-            "eslint-plugin/no-missing-message-ids": "error",
-            "eslint-plugin/no-unused-message-ids": "error",
-            "eslint-plugin/prefer-message-ids": "error",
```

Comment:
These three rules are now already in the rules recommended config, so I removed them from here.